### PR TITLE
[fix](profile) Fix ExecutionProfile is missing

### PR DESCRIPTION
**Pull request by zhiqiang-hhhh**
Make sure we do not eject profile of running query.

### CODE REVIEW

**Comment by zhiqiang-hhhh** [comment]
This fix need to be merged to 2.1 branch.

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
p0 失败与 pr 无关

File: fe/fe-core/src/main/java/org/apache/doris/common/util/ProfileManager.java [hunk 1]
```diff
@@ -178,6 +178,11 @@ public void addExecutionProfile(ExecutionProfile executionProfile) {
             if (queryIdToExecutionProfiles.size() > 2 * Config.max_query_profile_num) {
                 List<ExecutionProfile> finishOrExpireExecutionProfiles = Lists.newArrayList();
                 for (ExecutionProfile tmpProfile : queryIdToExecutionProfiles.values()) {
+                    // Query still running, skip them.
+                    if (tmpProfile.getQueryFinishTime() == ExecutionProfile.QUERY_FINISH_TIME_INIT) {
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/common/util/ProfileManager.java:182]: There maybe many unfinished profile in manager and FE will OOM.

**Pull request closed without merging**